### PR TITLE
feat: add Gallery Page

### PR DESCRIPTION
**Pull request by Jayu1214**
Gallery Page added ....
Screenshot 2024-03-19 182810

### CODE REVIEW

File: src/pages/Gallery/Gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  const goToPrevious = () => {
+    setSelectedImageIndex((prevIndex) =>
+      prevIndex === 0 ? images.length - 1 : prevIndex - 1,
+    );
+  };
```
Comment (codexsit) [inline src/pages/Gallery/Gallery.jsx:33]: check this logic, doesn't look good to me

File: src/pages/Gallery/Gallery.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  const goToNext = () => {
+    setSelectedImageIndex((prevIndex) =>
+      prevIndex === images.length - 1 ? 0 : prevIndex + 1,
+    );
+  };
```
Comment (codexsit) [inline src/pages/Gallery/Gallery.jsx:39]: same here

**Review by codexsit — DISMISSED** [review]
make necessary changes, as suggested

**Review by Achyut-0705 — APPROVED** [review]
LGTM!

**Pull request merged**